src/test/kotlin/org/malachite/estella/queue/InterviewDateTest.kt

```diff
@@ -0,0 +1,154 @@
+package org.malachite.estella.queue
+
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import org.junit.jupiter.api.*
+import org.malachite.estella.BaseIntegration
+import org.malachite.estella.aplication.domain.ApplicationRepository
+import org.malachite.estella.aplication.domain.ApplicationStageRepository
+import org.malachite.estella.commons.models.interviews.Interview
+import org.malachite.estella.commons.models.offers.Application
+import org.malachite.estella.commons.models.offers.ApplicationStageData
+import org.malachite.estella.commons.models.offers.ApplicationStatus
+import org.malachite.estella.commons.models.tasks.Task
+import org.malachite.estella.commons.models.tasks.TaskResult
+import org.malachite.estella.commons.models.tasks.TaskStage
+import org.malachite.estella.people.domain.JobSeekerRepository
+import org.malachite.estella.process.domain.RecruitmentStageRepository
+import org.malachite.estella.queues.utils.InterviewResultRabbitDTO
+import org.malachite.estella.task.domain.TaskRepository
+import org.malachite.estella.task.domain.TaskStageRepository
+import org.malachite.estella.util.DatabaseReset
+import org.malachite.estella.util.EmailServiceStub
+import org.springframework.amqp.core.Message
+import org.springframework.amqp.core.MessageProperties
+import org.springframework.amqp.rabbit.core.RabbitTemplate
+import org.springframework.beans.factory.annotation.Autowired
+import strikt.api.expectThat
+import strikt.assertions.isEqualTo
+import java.sql.Date
+import java.sql.Timestamp
+import java.time.Instant
+import java.time.LocalDateTime
+import java.util.*
+import javax.sql.rowset.serial.SerialBlob
+import javax.sql.rowset.serial.SerialClob
+
+@DatabaseReset
+@TestMethodOrder(MethodOrderer.OrderAnnotation::class)
+class InterviewDateTest : BaseIntegration() {
+    @Autowired
+    lateinit var rabbitTemplate: RabbitTemplate
+
+
+    private lateinit var task: Task
+    private lateinit var taskStage: TaskStage
+    private lateinit var applicationStageData: ApplicationStageData
+
+    @BeforeEach
+    fun prepareTask() {
```

Comment:
możliwe, że w TaskResultTest które ja pisałem jest to samo, ale funkcja nazywa się `prepareTask` a w środku w sumie nie ma tworzonego żadnego taska ani nawet taskStage xD

Reply:
Jest tak samo, bo chciałem jak najbardziej zachować konwencję wewnątrz systemu XDDD